tests/requirements_txt_fixer_test.py

```diff
@@ -50,6 +50,24 @@
             FAIL,
             b'Django\nijk\ngit+ssh://git_url@tag#egg=ocflib\n',
         ),
+        (
+            b'b==1.0.0\n'
+            b'c=2.0.0 \\ \n'
```

Comment:
whitespace after a backslash I believe is not valid for a contiuation? 🤔

Reply:
Ya, fixed test case